kpi/deployment_backends/kobocat_backend.py

```diff
@@ -1070,3 +1066,48 @@ def __validate_bulk_update_submissions(submissions: list) -> list:
             )
         return submissions
 
+    @staticmethod
+    def __prepare_bulk_update_response(kc_responses: list) -> dict:
+        '''
+        Formatting the response to allow for partial successes to be seen
+        more explicitly.
+
+        Args:
+            kc_responses (list): A list containing dictionaries with keys of
+            `_uuid` from the newly generated uuid and `response`, the response
+            object received from Kobocat
+
+        Returns:
+            dict: formatted dict to be passed to a Response object and sent to
+            the client
+        '''
+
+        # Unfortunately, the response message from OpenRosa is in XML format,
+        # so it needs to be parsed before extracting the text
+        results = [
+            {
+                'uuid': res['uuid'],
+                'status_code': res['response'].status_code,
+                'message': ET.fromstring(res['response']._content)
```

Comment:
why `_content` instead of `content`? the leading underscore is a Python convention—not enforced in any way—that indicates a private variable. since `_content` is coming from the Requests library and not our own code, we should probably keep our hands off unless we have good reason to touch it :)

Reply:
I wish I had a good reason for that — thanks for spotting. Sorted 👌